src/ocean/net/ssl/openssl/OpenSsl.d

```diff
@@ -0,0 +1,833 @@
+
+    public void SSL_CTX_set_verify (SSL_CTX* ctx, int mode,
+        int function (int, X509_STORE_CTX*) callback);
+
+
+    /***************************************************************************
+
+        Sets the maxiumum depth for certificate chain verification
+
+        Params:
+            ctx = the SSL context for which the depth should be set
+            depth = The maximum depth to be allowed
+
+    ***************************************************************************/
+
+    public void SSL_CTX_set_verify_depth (SSL_CTX* ctx, int depth);
+
+
+    /***************************************************************************
+
+        Specifies the locations for ctx, at which CA certificates for
+        verification purposes are located. The certificates available via
+        CAfile and CApath are trusted.
+
+        When looking up CA certificates, the OpenSSL library will first search
+        the certificates in CAfile, then those in CApath.
+
+        Params:
+            ctx = the CTX
+            CAfile = pointer to a file of CA certificates in PEM format, or
+                null. The file can containe several CA certificates.
+            CApath = a directory containing CA certificates in PEM format
+
+        Returns:
+            0 if the operation fails, 1 if the operation was successful
+
+    ***************************************************************************/
+
+    public int SSL_CTX_load_verify_locations (SSL_CTX* ctx,
+        Const!(char*) CAfile, Const!(char*) CApath);
+
+
+    /***************************************************************************
+
+        Get the result of peer certficate verification
+
+        Params:
+            ssl = the SSL object which obtained the peer certificate
+
+        Returns:
+            X509_V_OK if the verification succeeded or no peer certificate was
+            presented, ot an error code if the verification failed.
+
+    ***************************************************************************/
+
+    public c_long SSL_get_verify_result (/*const*/ SSL* ssl);
+
+
+    /***************************************************************************
+
+        Sets the list of available ciphers
+
+        Sets the list of available ciphers (TLS v1.2 and below). For TLSv1.3,
+        this function has no effect; call SSL_set_ciphersuites instead.
+
+        Params:
+            ssl = the SSL object to set the cipher list for
+            str = a colon-delimited sequence of cipher names
+
+        Returns:
+            1 if any cipher could be selected, 0 on complete failure
+
+    ***************************************************************************/
+
+    public int SSL_set_cipher_list (SSL* ssl, Const!(char*) str);
+
+
+    /***************************************************************************
+
+        Configure the available TLS v1.3 ciphersuites for an ssl object
```

Comment:
Wow, bleeding edge. Then you must be binding to v1.1.1, right ?

Reply:
Ugh. You're right.  Actually I never call either of these functions, I only ever used the default cipher list, so they don't need to be included at all.
I'll remove it from the bindings, since there is nothing else here which supports v1.1.x.